src/CPAL-1.0.xml

```diff
@@ -0,0 +1,643 @@
+<spdx name="Common Public Attribution License 1.0" identifier="CPAL-1.0" osi-approved="true">
+  <urls>
+    <url>http://www.opensource.org/licenses/CPAL-1.0</url>
+  </urls>
+  <header> The contents of this file are subject to the Common Public Attribution License Version 1.0 (the
+          &#x201C;License&#x201D;); you may not use this file except in compliance with the License. You may
+          obtain a copy of the License at _____. The License is based on the Mozilla Public License Version
+          1.1 but Sections 14 and 15 have been added to cover use of software over a computer network and
+          provide for limited attribution for the Original Developer. In addition, Exhibit A has been modified
+          to be consistent with Exhibit B. Software distributed under the License is distributed on an
+          &#x201C;AS IS&#x201D; basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+          License for the specific language governing rights and limitations under the License. The Original
+          Code is 
+    <alt name="code" match=".+">_____</alt>. The Original Developer is not the Initial Developer and is 
+    <alt name="developer" match=".+">_____</alt>. If left blank, the Original Developer is the Initial Developer.
+    The Initial Developer of the Original Code is 
+    <alt name="InitialDeveloper" match=".+">_____</alt>. All portions of the code written by 
+    <alt name="createdby" match=".+">_____</alt> are Copyright (c) 
+    <alt name="copyright" match=".+">_____</alt>. All Rights Reserved. Contributor 
+    <alt name="contributor" match=".+">_____</alt>. Alternatively, the contents of this file may be used under the
+    terms of the 
+    <alt name="AltLicense" match=".+">_____</alt> license (the 
+    <alt name="licenseName" match=".+">[____]</alt> License), in which case the provisions of 
+    <alt name="licenseName" match=".+">[____]</alt> License are applicable instead of those above. If you wish to
+    allow use of your version of this file only under the terms of the 
+    <alt name="licenseName" match=".+">[____]</alt> License and not to allow others to use your version of this file
+    under the CPAL, indicate your decision by deleting the provisions above and replace them with the notice and
+    other provisions required by the 
+    <alt name="licenseName" match=".+">[____]</alt> License. If you do not delete the provisions above, a recipient
+    may use your version of this file under either the CPAL or the 
+    <alt name="licenseName" match=".+">[____]</alt> License. 
+  </header>
+  <license>
+    <title>
+      <p>Common Public Attribution License Version 1.0 (CPAL)</p>
+    </title>
+    <body>
+      <list>
+        <li>
+          <b>1.</b>
+          <p>&#x201C;Definitions&#x201D;</p>
+          <p>1.0.1 &#x201C;Commercial Use&#x201D; means distribution or otherwise making the Covered Code
+             available to a third party.</p>
+        </li>
+        <li>
```

Comment:
Needs 1.1, 1.2 etc converting into nested lists, and similarly elsewhere in the document.